src/jit/codegencommon.cpp

```diff
@@ -7591,11 +7655,19 @@ void CodeGen::genProfilingLeaveCallback(unsigned helper /*= CORINFO_HELP_PROF_FC
         getEmitter()->emitIns_R_S(INS_lea, EA_PTRSIZE, REG_ARG_1, 0, 0);
     }
 
+#if !defined(UNIX_AMD64_ABI)
     // We can use any callee trash register (other than RAX, RCX, RDX) for call target.
     // We use R8 here. This will emit either
```

Comment:
Can you please add similar comment, but replacing R8 by R11 to the `#else` branch?